test-data/unit/stubgen.test

```diff
@@ -338,10 +338,24 @@ class A: ...
 class B(A): ...
 
 [case testDecoratedFunction]
+import x
+
 @decorator
 def foo(x): ...
+
+@x.decorator
+def bar(x): ...
+
+@decorator(x=1, y={"a": 1})
+def foo_bar(x): ...
```

Comment:
Decorator `CallExpr` are left for a future PR. For type checker to be able to infer the correct type, all arguments need to be passed along as well.